mtags/src/main/scala/scala/meta/internal/metals/ScaladocIndexer.scala

```diff
@@ -17,118 +17,135 @@ import scala.meta.pc.SymbolDocumentation
  */
 class ScaladocIndexer(
     input: Input.VirtualFile,
-    fn: SymbolDocumentation => Unit
+    fn: SymbolDocumentation => Unit,
+    occFn: Option[SymbolOccurrence => Unit]
```

Comment:
If we specify `occFn` to `ScalafmtIndexer`, do not calculate docstrings because this function is usually used for expiring the symbol.

Reply:
Why do you need to expire the symbol? Wouldn't we just be good with updating it?

Reply:
Och ok, we update the docstrings on demand, I didn't see it.